src/bika/lims/content/worksheet.py

```diff
@@ -1362,10 +1362,7 @@ def copy_src_fields_to_dst(src, dst):
             review_state = workflow.getInfoFor(analysis, 'review_state', '')
             if review_state in ['published', 'verified', 'retracted']:
                 old_ws_analyses.append(analysis.UID())
-
-                # XXX where does position come from?
-                old_layout.append({'position': position,
```

Comment:
`position` is not defined in this context and nobody cared for almost 10 years. so if this is a bug it seems to be minor. in any case it would be necessary to look deeper into the desired behavior here before being able to find a proper value `position` and I personally thing it's not worth the effort right now.